sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -3325,8 +3325,11 @@ object GpuOverrides extends Logging {
     expr[org.apache.spark.sql.execution.ScalarSubquery](
       "Subquery that will return only one row and one column",
       ExprChecks.projectOnly(
-        TypeSig.commonCudfTypes + TypeSig.NULL + TypeSig.DECIMAL_128,
-        TypeSig.commonCudfTypes + TypeSig.NULL + TypeSig.DECIMAL_128,
+        (TypeSig.commonCudfTypes + TypeSig.NULL + TypeSig.DECIMAL_128 + TypeSig.CALENDAR
+            + TypeSig.ARRAY + TypeSig.MAP + TypeSig.STRUCT)
+            .nested(TypeSig.commonCudfTypes + TypeSig.NULL + TypeSig.DECIMAL_128 +
+                        TypeSig.ARRAY + TypeSig.MAP + TypeSig.STRUCT),
```

Comment:
Curious, what is it about `CALENDAR` that makes it not supported as a nested type when every other type is supported?  Also if we support `CALENDAR` here, can we also support `BINARY`?

Reply:
Hi @jlowe, its my negligence. I've removed the `CALENBAR` here. Besides, I strengthened the tests.